agb/src/display/tiled/map.rs

```diff
@@ -88,6 +86,70 @@ where
         DISPLAY_CONTROL.get() & (1 << (self.background_id() + 0x08)) > 0
     }
 
+    fn commit(&mut self, vram: &mut VRamManager) {
+        let screenblock_memory = self.screenblock_memory() as *mut u8;
+
+        if *self.tiles_dirty() {
+            unsafe {
+                let tiledata: Vec<u8> = self.tiles_mut().iter().map(|a| a.tile_index(FourBpp).raw_index() as u8).collect();
```

Comment:
Should this be `EightBpp`? I know that it doesn't actually make a difference, but it feels more correct here :)

Could you also pull this line out of the unsafe block since it doesn't really need to go there

Reply:
Pushed it in the latest commit :)